src/components/navigation/mobile-menu.tsx

```diff
@@ -25,17 +30,19 @@ export function MobileMenu(props: MobileMenuProps) {
   )
 }
 
-// TODO: needs type declaration
-type EntryProps = any
+interface MobileMenuLink {
```

Comment:
I'd probably let the interface name match `ComponentNameProps` as long as there is no reason not to do that

Reply:
This uses [Declaration Merging](
https://www.typescriptlang.org/docs/handbook/declaration-merging.html). Is that a good enough reason?

Reply:
I wouldn't do that, especially not for React components. Instead:
```ts
interface EntryProps extends MobileMenuLink {
  childKey?: string
  isChild?: boolean
}
```
achieves the same and you don't need to know what there is another `MobileMenuLink` in scope that is also used.

Reply:
nice, did not find this one in the docs.